ext/src/ruby_api/func.rs

```diff
@@ -338,6 +338,30 @@ impl<'a> Caller<'a> {
         }
     }
 
+    /// (see Store#fuel_consumed)
+    /// @yard
```

Comment:
Putting `@yard` last so the `(see ...)` is first, which is a requirement of YARD to use ["reference tags"](https://rubydoc.info/gems/yard/file/docs/GettingStarted.md#reference-tags).

I'll fix this in YARD-rustdoc, but figured that workaround was acceptable.